packages/millicast-sdk/src/Publish.js

```diff
@@ -232,26 +237,27 @@ export default class Publish extends BaseWebRTC {
 
     const workerBlob = new Blob([workerString])
     const workerURL = URL.createObjectURL(workerBlob)
-    const worker = new Worker(workerURL)
+    this.worker = new Worker(workerURL)
 
-    const senders = this.getRTCPeerConnection()
-      .getSenders()
-      .filter((elt) => elt.track.kind === 'video')
-    const sender = senders[0]
+    const senders = this.getRTCPeerConnection().getSenders()
```

Comment:
What about audio tracks? Do we not need to filter them out anymore?

Reply:
Oh is that because of the no audio bug reported on slack?

Reply:
Exactly, we need to create all tracks into encoded streams to be seen/heard